core/domain/acl_decorators.py

```diff
@@ -561,5 +611,36 @@ def test_can_modify(self, exploration_id, **kwargs):
         raise base.UserFacingExceptions.UnauthorizedUserException(
             'You do not have credentials to change rights for this '
             'exploration.')
+    test_can_modify.__wrapped__ = True
 
     return test_can_modify
+
+
+def can_perform_cron_tasks(handler):
+    """Decorator to ensure that the handler is being called by cron or by a
+    superadmin of the application.
+    """
+
+    def can_perform(self, **kwargs):
```

Comment:
Standardize the naming of these inner functions please. They can all start with "test" or all start with "can" but don't mix the two.